utils_nlp/models/xlnet/utils.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+from matplotlib import pyplot
+from sklearn.metrics import confusion_matrix
+import seaborn as sn
+import numpy as np
+import pandas as pd
+def generate_confusion_matrix(y_true, y_pred,labels):
```

Comment:
this function could be reused by other parts of the code, maybe can go on this folder: https://github.com/microsoft/nlp/tree/master/utils_nlp/eval

Reply:
that's a good idea, will fix on the next commit!

Reply:
@agupte93